pkg/apis/catalog/v1alpha1/derivedcustomresource_types.go

```diff
@@ -184,6 +184,9 @@ func (s *DerivedCustomResource) IsReady() bool {
 	if s.Generation != s.Status.ObservedGeneration {
 		return false
 	}
+	if !s.DeletionTimestamp.IsZero() {
```

Comment:
This fix forbids creating new derived custom resources if the `DerivedCustromResource` CRD instance is under deletion.